### DISCUSSION TITLE
Call home or cloud based

### DISCUSSION THREAD

**Discussion by peter-a-w** [freeplane/freeplane#2153] [Help]
Dear all,

I'm new to Freeplane and out of security concerns I would like to know if Freeplane requires internet connection, is cloud based, or has a call home function?

If not, will future updates also avoid this?

Kind Regards, 

Peter

**Comment by peter-a-w** [comment]
In other words, Can someone external access the data? Is the data I enter only stored locally?

**Comment by euu2021** [comment]
The data is only stored locally. But, be aware of this: https://github.com/freeplane/freeplane/discussions/752

The relevant setting is: `Preferences…->Environment->Automatic bug report->Policy`
